client/src/components/common/applicant-info-1.vue

```diff
@@ -135,7 +135,7 @@
             </v-col>
           </v-row>
 
-          <v-row class="mt-2" v-if="applicant.addrLine1 && !showAddressMenu">
```

Comment:
I agree that we should just show this (since it was auto-displayed after the address search anyway).